### DISCUSSION TITLE
Schism Tracker 20241021

### DISCUSSION THREAD

**Discussion by mrpapersonic** [schismtracker/schismtracker#544] [Announcements]
Summary of changes since the [20240909](https://github.com/schismtracker/schismtracker/releases/tag/20240909) release:
* Fix search on the Load Module page
* Fix resizing samples with interpolation
* Wii U support
* Remove the reliance on memory-mapped files, allowing for less memory usage on systems without virtual memory (such as the Wii)
* Fix portamento in both volume and effect column at the same time
* Allow importing of IEEE floating point WAV and AU files
* POSIX: Use `~/.config/schism` by default; `~/.schism` is used if it is available as a fallback
* Haiku OS: Use `~/config/settings` by default, and fallback to `~/.schism` if it's available
* MIDI/JACK: Both midi in and out should work flawlessly now
* S3M: Fix sample import under big endian architectures
* IT: Fix V81-VFF commands in old Schism Tracker ITs
* IT: Correctly fingerprint IT2.14 patch versions
* IT: Drop SAx effect from IT 1.xx files
* XM: Import ModPlug Tracker Command X extensions

<hr /><em>This discussion was created from the release <a href='https://github.com/schismtracker/schismtracker/releases/tag/20241021'>Schism Tracker 20241021</a>.</em>

**Comment by brandflake11** [comment]
Hello, I would like to just say thank you for the awesome features in this release. This may be one of the most exciting releases yet for schism for me, as now 24 and 32-bit wav files can be loaded. This just doubled the files I can natively load in schism without having to convert the samples manually (or with a script).

Not only that, but now there's alsa midi support? This is amazing. Keep it up! :)

**Comment by nertzy** [comment]
I noticed on the [release page](https://github.com/schismtracker/schismtracker/releases/tag/20241021) that `schismtracker-20241021-macos.zip` was published (re-published?) on October 30.

The rest of the release was published on October 21. Is this expected?

When upgrading via Homebrew, I'm getting the following related failure:

```
❯ brew install --verbose schism-tracker
==> Upgrading 1 outdated package:
schism-tracker 20240909 -> 20241021
==> Upgrading schism-tracker
==> Downloading https://github.com/schismtracker/schismtracker/releases/download/20241021/schismtracker-20241021-macos.zip
Already downloaded: /Users/grant/Library/Caches/Homebrew/downloads/433f703063ebbdd3e53dadb9c14bf014a62004173dd47bbf4d6068b60fc3ca67--schismtracker-20241021-macos.zip
==> Verifying checksum for '433f703063ebbdd3e53dadb9c14bf014a62004173dd47bbf4d6068b60fc3ca67--schismtracker-20241021-macos.zip'
==> Purging files for version 20241021 of Cask schism-tracker
Error: schism-tracker: SHA256 mismatch
Expected: 61549e3d5088f1c32da2f08642810ee7653e6d96e057ca935d932bbdb0d18f0b
  Actual: 0ddf88b6196fc32337c9395ec07c839d84d68b75830eefd42029d763a3748837
    File: /Users/grant/Library/Caches/Homebrew/downloads/433f703063ebbdd3e53dadb9c14bf014a62004173dd47bbf4d6068b60fc3ca67--schismtracker-20241021-macos.zip
To retry an incomplete download, remove the file above.
```

**Reply by mrpapersonic** [reply]
yes, I fixed an issue with the release where the path of utf8proc was incorrect on powerpc, but the actual contents of the binary have not changed, hence no new release. I'll put up a notice on the releases page if I end up doing this again in the future :)

**Reply by nertzy** [reply]
OK thanks for the confirmation, we can get the SHA256 updated in Homebrew then.

**Reply by nertzy** [reply]
https://github.com/Homebrew/homebrew-cask/pull/195545
